examples/quickstart/jupyter-notebooks/kafka-tutorial.ipynb

```diff
@@ -0,0 +1,753 @@
UKlUmh37o0ePkJ2djebNm6NRo0ZwcHDAxIkTsWDBAkydOhXr1q3DG2+8gWnTpuHvv//W6nXGM2cyVOrv/xdffIG33noLO3bsAAD8+OOPCA0NxbfffoumTZvim2++gUKhQK1atZCRkQF3d3cmRzpk8DVIRGScPvroI+zZswc3btxA5cqV8dtvv6FmzZqa57/55htMnDgRU6ZMwYQJE1C+fHkeHMhoKJVK9O3bF5s2bYK1tTXGjh2L3r17Y9OmTbh9+zZmzpyJqlWrYvfu3di0aRNWrFjBkwIdY4JERCVC3oLsX375Be+99x6mTp2KW7duYcWKFRg5ciRGjx6NypUra17z+eef46+//tLqls/kiIzFgQMHsHbtWjRp0kRzgvDo0SOcPn0akyZNwsiRI7Xq7FiQrVtMkIioRDl06BB+++03NG7cWNMdeOnSpZg9ezb69++Pd999VytJyluvxOSIDN2CBQsgInjvvfegUqkwdOhQKBQKfPfdd9iwYQOOHDmC77//HgBw/vx51K1bV88RGy9WMBJRiXH//n0MGTIEDx48QI0aNTTLR44cCRHBnDlzYGpqiiFDhqBq1aoAwOSIjEZ2djbS0tIwffp0nDlzBkOGDMHKlSvRqFEjLFy4EJMmTULfvn1hY2ODy5cvw9fXV98hGzWOIBFRiXLhwgW8/vrrqFy5Mr766iutM+Rly5ZhzJgxWLx4MUaMGKHHKImKzuXLlzFt2jRERUWhdu3aaNu2LbZu3YqpU6ciICAAwL8jp7ysVnSYIBFRiXP+/Hm8/fbbaNiwIcaNG4fatWtrntu8ebOmzxGRsYqNjcWRI0fwxRdf4MKFC7CxscH48ePx8ccfa9bhyGnRYoJERCVScHAwhg4digYNGmD8+PH5LifwzJlKi48//hhff/01GjdujAMHDug7nFKDCRIRlVjBwcEYPnw4KleujHnz5sHLy0vfIREVm7wjRKdPn0aDBg1gamrKkaNiYjSNIonI+Pj7+2Px/9u7v5Cm3jgM4M88lmx55mg4bdF2UDyLgjX7NyJKUEHJFgsSAkNI6iKTIIrCwlxERcpQIggrchddLCiEwgwySIJ1EUEyCApWTKjEbkJGZXV8u/jhoXP89fujZouez9Xe79n5nvecq+f82y5cgCzLhjfXiP4EUy8gAMD69eshSRI0TWM4mie8gkREWW/qjPn730oiIvqZGJCI6LfA2wpENJ94KkZEvwWGIyKaTwxIRERERCYMSEREREQmDEhEREREJgxIRERERCYMSEREREQmDEhEREREJgxIRERERCYMSET0x9I0DZOTk796GkSUhRiQiCirKIqC7u5uQy0QCCASiUAIgUgkAo/Hg7y8PLjdbhw4cED/3sTEBA4fPoylS5di0aJFCAaDePDggb48FovB4XDg1q1bWLFiBfLy8jAyMgJFUXDmzBk0NTVBlmV4PB5cunTJMIejR49CVVXYbDaUlJSgra0NX7580ZdHIhEEAgFcvXoVHo8H+fn5aG5uhqZp6OjoQHFxMVwuF06fPm3o+/79e+zZsweFhYWw2+2orKzE8PDw3B1QIpqR3F89ASKi/+rmzZvo6upCPB7HypUrMTo6aggTLS0tePbsGeLxONxuN/r6+lBbW4tkMomysjIAwIcPH3Du3DlcuXIFTqcTLpcLABCNRnHq1CkcO3YMN27cwL59+1BRUQGfzwcAkGUZsVgMbrcbyWQSe/fuhSzLOHLkiL79VCqFgYEB3L17F6lUCjt27MDLly+hqiqGhoaQSCTQ1NSE6upqBINBAEB9fT2sVisGBgZQUFCAnp4eVFVV4cWLF1i8ePF8HVoiMhNERFnE6/WKrq4uQ23VqlWivb1dRKNRoaqq+Pz587T10um0kCRJvH792lCvqqoSra2tQgghent7BQDx9OnTadvctWuXPp6cnBQul0tcvHjxh/Ps7OwUa9as0cft7e3CZrOJ8fFxvVZTUyMURRGapuk1n88nzp49K4QQ4uHDh8Jut4tPnz4ZepeWloqenp4fbpuIfj5eQSKi30Z9fT26u7tRUlKC2tpabNmyBaFQCLm5uUgmk9A0DaqqGtaZmJiA0+nUxwsXLoTf75/W+/uaxWJBcXExxsbG9Nr169dx/vx5pFIpZDIZfP36FXa73dBDURTIsqyPi4qKIEkScnJyDLWpvsPDw8hkMob5AcDHjx+RSqX+z6EhojnGgEREWSUnJwdCCENt6lmfZcuW4fnz5xgcHMS9e/fQ3NyMzs5ODA0NIZPJQJIkPHnyBJIkGdbPz8/XP1ut1r/949sFCxYYxhaLRX+A+9GjR2hoaMDJkydRU1ODgoICxONxRKPRf+3xT30zmQyWLFlieE5qisPhmFYjovnDgEREWaWwsBBv377Vx+Pj43j16pU+tlqtCIVCCIVC2L9/P5YvX45kMony8nJomoaxsTFs2rRpTueUSCTg9Xpx/PhxvZZOp2fdd/Xq1RgdHUVubi4URZl1PyKaOwxIRJRVKisrEYvFEAqF4HA4cOLECf2KUCwWg6ZpCAaDsNlsuHbtGqxWK7xeL5xOJxoaGtDY2IhoNIry8nK8e/cO9+/fh9/vR11d3YznVFZWhpGREcTjcaxbtw79/f3o6+ub9b5WV1djw4YNCIfD6OjogKqqePPmDfr7+7F9+3asXbt21tsgopnha/5ElFVaW1tRUVGBrVu3oq6uDuFwGKWlpQD+uu10+fJlbNy4EX6/H4ODg7h9+7b+DE9vby8aGxtx6NAh+Hw+hMNhPH78GB6PZ1Zz2rZtGw4ePIiWlhYEAgEkEgm0tbXNel8tFgvu3LmDzZs3Y/fu3VBVFTt37kQ6nUZRUdGs+xPRzFmE+WY/ERER0R+OV5CIiIiITBiQiIiIiEwYkIiIiIhMGJCIiIiITBiQiIiIiEwYkIiIiIhMGJCIiIiITBiQiIiIiEwYkIiIiIhMGJCIiIiITBiQiIiIiEy+AQnBnCvIWtqFAAAAAElFTkSuQmCC\n",
```

Comment:
For me this is showing 1 post for everybody, so the correlation bit doesn't come through that much.
<img width="870" alt="Screen Shot 2023-04-13 at 10 26 26 AM" src="https://user-images.githubusercontent.com/38529548/231837863-320962be-7b76-4dc4-bc18-8969c30956fb.png">

Reply:
I think it's due to the scale 😬 The LHS axis is multiplied by 10 million and the RHS is multiplied by 1 million, so it's hard to see differences. I can try to play around with the axis limits for the bar plots so the differences are more pronounced.

Reply:
I configured the bar plot y-axis to log scale which shows the trends much better:
![image](https://user-images.githubusercontent.com/7747997/235805862-a5ca7378-09ff-43ff-965f-2229173203f6.png)

Reply:
I configured the bar plot y-axis to log scale which shows the trends much better:

![image](https://user-images.githubusercontent.com/7747997/235806145-61330794-13a2-4a1e-801b-79675549d4f0.png)